### PR TITLE
[BPK-4390]: Migrate components L-P to new format

### PR DESCRIPTION
**Pull request by olliecurtis**
Migrate all components from L to P to new stories format

Tweaked input examples for use on docs site

Remember to include the following changes:

- [ ] `UNRELEASED.md`
- [ ] `README.md`
- [ ] Tests
- [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: packages/bpk-component-link/examples.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+const ButtonLinkExample = () => (
+  <div>
+    <BpkButtonLink onClick={action('#1 clicked')}>Link #1</BpkButtonLink>
+    <br />
+    <BpkButtonLink onClick={action('#2 clicked')}>Link #2</BpkButtonLink>
+  </div>
+);
+
+const LinkAlternativeExample = () => (
+  <div style={{ backgroundColor: colorSkyGrayTint01, padding: spacingBase }}>
```
Comment (shaundon) [inline packages/bpk-component-link/examples.js:52]: Can we use `BpkDarkExampleWrapper` for this instead?

File: packages/bpk-component-link/examples.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    </BpkLink>
+    <br />
+    <BpkLink href="#" onClick={action('#2 clicked')} alternate>
+      Link #2
+    </BpkLink>
+  </div>
+);
+
+const ButtonLinkAlternativeExample = () => (
+  <div style={{ backgroundColor: colorSkyGrayTint01, padding: spacingBase }}>
```
Comment (shaundon) [inline packages/bpk-component-link/examples.js:64]: Can we use `BpkDarkExampleWrapper` for this instead?

File: packages/bpk-component-nudger/examples.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+  //     min={1}
+  //     max={99}
+  //     value={3}
+  //     onChange={action('change')}
+  //     decreaseButtonLabel="Decrease"
+  //     increaseButtonLabel="Increase"
+  //     buttonType="outline"
+  //   />
+  // </div>
+);
```
Comment (shaundon) [inline packages/bpk-component-nudger/examples.js:194]: Looks like this was left in by accident?

**Review by shaundon — CHANGES_REQUESTED** [review]

File: packages/bpk-component-link/examples.js (hunk 1 above)
Comment (olliecurtis) [reply packages/bpk-component-link/examples.js:52]: Sure, I tried to do it but the text got unreadable so will have a play and tweak with it

File: packages/bpk-component-link/examples.js (hunk 2 above)
Comment (olliecurtis) [reply packages/bpk-component-link/examples.js:64]: Same as above

File: packages/bpk-component-nudger/examples.js (hunk 3 above)
Comment (olliecurtis) [reply packages/bpk-component-nudger/examples.js:194]: Yup!

**Review by shaundon — APPROVED** [review]

**Pull request merged**